### PR TITLE
[#2] [Chore] Review readme and PR templates

### LINKED ISSUE #2: [Chore] Update Readme and PR templates
## Why

In order to start from clean foundations, ensure the 'README.md' and the default PR templates are aligned with the project need.
 
## Who Benefits?

**Developers** and **reviewers** will have a clean repository even before we bring code in.

### PR DESCRIPTION
**Pull request by malparty**
`N/A`

## What happened 👀

Define a base for the `README.md` and fix minor details in the PR templates.
 
## Insight 📝

`N/A`

## Proof Of Work 📹

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/chore_template.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+about: "Open a Chore for a minor update."
+title: "[Chore] "
+labels: "type: chore"
 ---
 
 ## Why
 
-Describe the update details and why it's needed. 
- 
+Describe the update details and why it's needed.
```
Comment (carryall) [inline .github/ISSUE_TEMPLATE/chore_template.md]: How about this? to align with the feature template

```suggestion
Describe the update details and why it is needed.
```

**Review by carryall — APPROVED** [review]
The rest LGTM 🙌🏼  @malparty please also move this PR to `Review in Progress`
Screen Shot 2564-10-12 at 10 37 12

**Comment by carryall** [comment]
> The rest LGTM 🙌🏼 @malparty please also move this PR to `Review in Progress`

Ahh just saw that you use the automation on the project lane after I approved the PR it goes to `Reviewer approved` 🤔  so should we set the minimum approval to 2 so that it's easier to tell which PRs is ready to merge

**Review by rafayet-monon — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/chore_template.md (hunk 1 above)
Comment (malparty) [reply .github/ISSUE_TEMPLATE/chore_template.md]: Nice catch :) 
✅ Fixed in 7c25d6

**Comment by malparty** [comment]
> > The rest LGTM 🙌🏼 @malparty please also move this PR to `Review in Progress`
> 
> Ahh just saw that you use the automation on the project lane after I approved the PR it goes to `Reviewer approved` 🤔 so should we set the minimum approval to 2 so that it's easier to tell which PRs is ready to merge

That should be OK, we will test for the next PR :) Thanks @carryall :)

**Pull request merged**